docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Documentation
```

Comment:
I would rename this to `Examples`. 

If later we add things here that are not examples we can change it accordingly. I think `Documentation` is too generic, since this page is about documentation, this is not adding significant information.